sw/host/opentitanlib/src/app/mod.rs

```diff
@@ -768,9 +705,8 @@ impl TransportWrapper {
     /// Also configure SPI port mode/speed, and other similar settings.
     pub fn apply_default_configuration(&self) -> Result<()> {
         self.transport.apply_default_configuration()?;
-        self.apply_pin_configurations(&self.pin_conf_map)?;
```

Comment:
It seems that this line has been lost in the refactor.  Applying configuration (push-pull, pull-up, default level) to each HyperDebug pin is an essential to it working.

Do we not have any CI tests that use the swizzle board on CW310?